src/components/variables.css

```diff
@@ -92,4 +92,12 @@
   --screen-sm-min: 768px;  /* Small screen / tablet */
   --screen-md-min: 992px;  /* Medium screen / desktop */
   --screen-lg-min: 1200px; /* Large screen / wide desktop */
+
+  /*
+   * Colors
+   * ======================================================================== */
+  --nyaaya_blue: rgb(232, 250, 255);
+  --nyaaya_green: rgb(84, 241, 178);
```

Comment:
Don’t describe colours in variable names. Refer to them as primary(green) and secondary(blue)